pkg/kwokctl/runtime/cluster.go

```diff
@@ -207,6 +207,25 @@ func (c *Cluster) KubectlInCluster(ctx context.Context, stm utils.IOStreams, arg
 		append([]string{"--kubeconfig", utils.PathJoin(conf.Workdir, InHostKubeconfigName)}, args...)...)
 }
 
+func (c *Cluster) EtcdctlInCluster(ctx context.Context, stm utils.IOStreams, args ...string) error {
+	conf, err := c.Config()
+	if err != nil {
+		return err
+	}
+
+	etcdctlPath, err := exec.LookPath("etcdctl")
+	if err != nil {
+		bin := utils.PathJoin(conf.Workdir, "bin")
+		etcdctlPath = utils.PathJoin(bin, "etcdctl"+vars.BinSuffix)
+		err = utils.DownloadWithCacheAndExtract(ctx, conf.CacheDir, conf.EtcdBinaryTar, etcdctlPath, "etcdctl"+vars.BinSuffix, 0755, conf.QuietPull, true)
+		if err != nil {
+			return err
+		}
+	}
+
+	return utils.Exec(ctx, "", stm, etcdctlPath, args...)
```

Comment:
For binary runtime, just specify the port of etcd, 
For kind runtime, may need to expose etcd's port to local with `kubectl port-forward` and use it
For docker runtime, also need to expose etcd's port to local, may use `docker exec -i ${etcdpodname} nc 127.0.0.1 ${etcdport}` can be work.

Reply:
It has been modified, thanks for your guidance. I have considered that kind is not supported at the beginning, so I removed the parameter. If kubectl port-forward is used, I can understand it no problem

Reply:
Add a test, Try to get a value using `kwokctl etcdctl` in this

https://github.com/kubernetes-sigs/kwok/blob/96f9165f68a0d71c587b32f935397665638985fc/test/kwokctl/kwokctl_workable_test.sh#L49

Reply:
By the way I have an idea, for `kind` and `docker` you can call the `etcdctl` directly from the container like `snapshot`.

Reply:
https://github.com/kubernetes-sigs/kwok/blob/ca4b082cec3bda5083b5080cbaf74803266583e2/pkg/kwokctl/runtime/compose/cluster_snapshot.go

https://github.com/kubernetes-sigs/kwok/blob/a98d86be2828090a7127d2033b7407bea111a40e/pkg/kwokctl/runtime/kind/cluster_snapshot.go

Reply:
EtcdctlInCluster should be implemented independently in each runtime's directory, as it is not universal to each other

Reply:
@Fish-pro

Reply:
@wzshiming Has been implemented separately at each runtime, the test script has a path problem, can I submit it in another pr